content/en/user-guide/aws/opensearch/index.md

```diff
@@ -154,7 +154,7 @@ $ curl http://my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566
 Or the health endpoint:
 
 {{< command >}}
-$ curl -s http://my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566/_cluster/health | jq .
+$ curl -s http://my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566/_cluster/_localstack/health | jq .
```

Comment:
These URLs are actually the health endpoints of OpenSearch, not LocalStack. So these endpoints don't change.